readme.md

```diff
@@ -0,0 +1,46 @@
+# Reddit jsAPI example consumer extension.
+
+This repository provides example code for the implementation of jsAPI consumers. When loaded in the browser it will do the following things:
+
+- Output all jsAPI events to the browser console.
+- Display the location of jsAPI containers on the page in the following format `⚠ type`. 
+- The container elements will have the payload data stringified as the title attribute. 
+
+The jsAPI specific code can be found here:
+
+`/src/inject/inject.js`
+
+
+# What is this jsAPI?
+
+jsAPI is shorthand for "*Javascript API for Third Party Tools*".
+
+##  Introduction 
+
+The redesign of reddit means that third party extensions and userscripts will no longer work and need to be rebuild as well. This rebuilding and integration in the redesigned reddit cannot be done with previous used methods because the following reasons: 
+
+1. **CSS classnames and IDs are no longer static.** They are instead computed by the style engine, not manually assigned by devs. Meaning they can and will change on each deploy. 
```

Comment:
🔪 `and IDs` -- only the class names are provided by the style engine.

Reply:
s/meaning they can and will change on each deploy/Class names may change when code affecting those components is deployed, and new code is deployed many times a day.